content/calloop-v0.7.markdown

```diff
@@ -0,0 +1,121 @@
+Title: Version 0.7 of calloop
+Date: 2020-10-09 19:00
+Category: Releases
+Slug: calloop-v-0-7
+Authors: Victor Berger
+Summary: Announcement of version 0.7 of calloop, and a presentation of the crate.
+
+We just released [version 0.7](https://github.com/Smithay/calloop/releases/tag/v0.7.0) of
+[calloop](https://docs.rs/calloop), the callback-based event loop that backs
+[Smithay](https://github.com/Smithay/smithay) and optionnaly
+[SCTK](https://github.com/Smithay/client-toolkit). As the crate is now getting mature, it is time
+for a formal presentation!
+
+## Callbacks, in Rust?
+
+It is commonly acknowledged that in Rust, callback-based designs induce a lot of friction, in
+particular regarding state sharing. Nobody wants to have `Rc<RefCell<_>>` littered all around their
+codebase, and so naturally most of the ecosystem turned to async/await, a design much more natural
+given the constraints of the language.
+
+However, not all usecases play well with async/await, notably cases were you are in a purely
+reactive context: waiting for events and quickly reacting to them. Indeed, why would you use
+async/await if you never need to `.await` anything? This is for a large part our context of
+work in Smithay, and so we developped calloop.
+
+Calloop is designed trying to ease the friction with Rust as much as possible. The central
+issue, state sharing, is managed though some global shared data. When you're running your event
+loop, you can provide the method with a `&mut Data` reference, that is then passed down to all
+of your callbacks. If this `Data` contains the state of your app, all your callbacks can then
+access it as if they were all methods of the same object.
+
+In a nutshell, calloop is mostly targetted at apps that spend a large portion of their time just
+waiting for something to happen, and need to quickly react to it. As opposed to apps that need
+to manage numerous computations or sockets in parallel efficiently, which is the more common
+use of async/await. This processing can be suited to GUI apps notably, in particular on X11 or
```

Comment:
'can be suited to' seems weird (not sure if incorrect though), I'd expect 'is suitable for' or something entirely different.